src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
I think there is no typo here.
The first check test if `lastActive` is not null or undefined and then check for the second condition and if the second pass, the method will return `lastActive`.

So this method return either `false` or `lastActive` (`Element`)